nearlib/README.md

```diff
@@ -1,22 +1,90 @@
-Near lib
-====
+# Near lib
 
 Near lib is a javascript library for development of dapps on near.
 
 ## Install dependencies
-
 ```
 npm install
 ```
-
 ## Test
 
 Start the node by following instructions from nearcore/README.md, then
-
 ```
 npm test
 ```
-
 ## Integration test
 
-Contract "hello.wasm" source code location: https://studio.nearprotocol.com/?f=mfaccXt
+Contract "hello.wasm" source code location: <https://studio.nearprotocol.com/?f=mfaccXt>
+
+## documentation
+To regenerate documentation, run
+```
+documentation readme near.js signing/key_pair.js -f md --shallow --section=API
```

Comment:
can we just put this into `package.json` so that you can `npm run docs` or `yarn docs`?

Reply:
yeah, it's coming up soon on my to-do list

Reply:
done